### ISSUE TITLE
Placeholder text has insufficient color contrast

### ISSUE THREAD

**Issue by arethsu** [JohnXLivingston/peertube-plugin-livechat#710]
**Description of issue**

This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 1.4.3 Contrast (Minimum) (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum).

Good contrast between text and background is important for readability, especially for people with impaired vision or while browsing in bright environments.

The placeholder text in all the chat input fields has insufficient contrast levels, like for example in the message box:

![Color contrast checker indicating insufficient contrast on the chat message text field placeholder](https://github.com/user-attachments/assets/776bf893-997e-4a64-ad36-cdfd63f96303)

Some fields change their appearance when selected, but still have insufficient contrast. Like in the emoji search field:

![Color contrast checker indicating insufficient contrast on the selected search emoji text field placeholder](https://github.com/user-attachments/assets/8e1d3ba4-2a15-40ba-92dd-e053fa5c88a2)

**To Reproduce**

1. Use a color contrast checker on input field placeholders (chat message input, emoji search input)

**Recommended solution**

Change the color of text or background so that the color contrast is at least 4.5:1.

For large text (24px, or 19px and bold), at least 3:1 applies.

There are many tools available for checking contrast levels, for example the one at https://contrast.report/

**Server:**

 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: TBA

**Desktop:**

 - OS: macOS 15.5
 - Browser: Safari 18.5

**Additional context**

This issue was part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), under assignment of [HowlRound Theatre Commons](https://howlround.com/).
